xmltest.cpp

```diff
@@ -1641,6 +1641,20 @@ int main( int argc, const char ** argv )
 		}
 	}
 
+	{
+		// Oh those memory leaks.

```

Comment:
Clearly this should be documented better - what where and why was leaking and the fact that it shouldn't leak now.